charts/kong/README.md

```diff
@@ -253,6 +253,8 @@ section of `values.yaml` file:
 | image.tag                          | Version of the ingress controller                                                     | 0.7.0                                                                        |
 | readinessProbe                     | Kong ingress controllers readiness probe                                              |                                                                              |
 | livenessProbe                      | Kong ingress controllers liveness probe                                               |                                                                              |
+| serviceAccount.create              | Create Service Account for IngresController                                           | true
+| serviceAccount.name                | Use existing Service Account, specifiy it's name                                      | ""
```

Comment:
"its" rather than "it's" here: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/its-vs-its